stable/pomerium/templates/proxy-deployment.yaml

```diff
@@ -113,15 +119,38 @@ spec:
             scheme: HTTPS
         resources:
 {{ toYaml .Values.resources | indent 10 }}
-{{- if or .Values.config.existingConfig .Values.config.policy }}
         volumeMounts:
+{{- if or .Values.config.existingConfig .Values.config.policy }}
         - mountPath: /etc/pomerium/
           name: config
+{{- end }}
+{{- /* TODO in future: Remove legacy logic */ -}}
+{{- if not .Values.existingLegacyTLSSecret }}
+        - mountPath: /pomerium/cert.pem
+          name: service-tls
+          subPath: {{ template "pomerium.proxy.tlsSecret.certName" . }}
+        - mountPath: /pomerium/privkey.pem
+          name: service-tls
+          subPath: {{ template "pomerium.proxy.tlsSecret.keyName" . }}
+        - mountPath: /pomerium/ca.pem
+          name: ca-tls
+          subPath: {{ template "pomerium.caSecret.certName" . }}
+{{- end }}
       volumes:
+{{- if or .Values.config.existingConfig .Values.config.policy }}
       - name: config
         configMap:
           name: {{ $configName }}
 {{- end }}
+{{- /* TODO in future: Remove legacy logic */ -}}
+{{- if not .Values.existingLegacyTLSSecret }}
```

Comment:
@travisgroth you forgot this one I think

Reply:
I clearly hadn't had enough coffee.  thanks.